code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -340,16 +340,32 @@
 	if(hasHUD(user,"security"))
 		var/perpname = get_visible_name(TRUE)
 		var/criminal = "None"
+		var/commentLatest = "None"
 
 		if(perpname)
 			for(var/datum/data/record/E in data_core.general)
 				if(E.fields["name"] == perpname)
 					for(var/datum/data/record/R in data_core.security)
 						if(R.fields["id"] == E.fields["id"])
 							criminal = R.fields["criminal"]
+			if(commentLatest)

```

Comment:
This does nothing since itll always be defined as "None" here

Reply:
I figured to define it that way, as of it does provide the "None" message if there is no datacore entry for the examined person, that is, it is unknown or not-crew. I guess it works so, that if the list macros return 0, as of the list in question is not existing, it will state the defined string. But if there is the list, it will return either latest comment or no entries in case of list existing, but it is empty. Comment log is defined at datacore.dm.

Could change the "None" to be more informative though, like "ERROR: Unable to locate a data core entry for this person", which is the message when you try to view comment log of a unknown person.